tracer/src/Datadog.Trace/Logging/Internal/DatadogLogging.cs

```diff
@@ -218,5 +221,38 @@ internal static string GetLogDirectory()
 
             return logDirectory;
         }
+
+        internal static void CleanLogFiles(string logsDirectory)
+        {
+            var logDaysLimit = EnvironmentHelpers.GetEnvironmentVariable(ConfigurationKeys.LogFileRetentionDays);
+            if (int.TryParse(logDaysLimit, out var days) && (days > 0 && days < int.MaxValue))
+            {
+                var date = DateTime.Now.AddDays(-1 * days);
+                var logFormats = new[]
+                  {
+                    "dotnet-tracer-managed-*-.log",
+                    "dotnet-tracer-native-*-.log",
+                    "dotnet-native-loader-*.log",
+                    "DD-DotNet-Profiler-Native-*.log"
+                  };
+                try
```

Comment:
super small nit: I'm actually surprised you didn't get stylecop warnings, but having the extra space between these two elements improves readability

```suggestion

                try
```

Reply:
Yeah, weird that I went back in to check it out and still no errors, fixed!